.gitlab-ci.yml

```diff
@@ -37,7 +37,7 @@ stages:
     include:
       - local: '.gitlab/custom-jobs.yml'
       - project: 'radiuss/radiuss-shared-ci'
-        ref: v2022.06.0
+        ref: task/remove-intel-18
```

Comment:
```suggestion
        ref: v2022.06.1
```

Reply:
I already applied the required change.

Reply:
Yup, saw after I did that, planning to remove this in a sec.  Thanks BTW!